lib/puppet/provider/service/debian.rb

```diff
@@ -63,4 +63,19 @@ def enable
     update_rc "-f", @resource[:name], "remove"
     update_rc @resource[:name], "defaults"
   end
+
+  def statuscmd
+    os = Facter.value(:operatingsystem).downcase
+    majversion = Facter.value(:operatingsystemmajrelease)
+
+    if ((os == 'debian' && majversion == '8') || (os == 'ubuntu' && majversion == '15.04'))
```

Comment:
@kylog re. merging, I was thinking I should make this apply to future debians and ubuntus as well, was going to update, though we could also do that when they are released.